src/routes.ts

```diff
@@ -143,7 +147,7 @@ const BottomTabNavigator = createBottomTabNavigator(BottomTabBarRoutes, {
   tabBarComponent: BottomBar,
 })
 
-const RegistrationScreens = createSwitchNavigator(
+const RegistrationScreens = createAnimatedSwitchNavigator(
```

Comment:
this is going to require some hardcore (re-)typing..... I thought this would came back and bite us some time, but yeahhhh...
anyway I think it should be something to be done later, in a different PR. I'm goign to `@ts-ignore` most things